utils/tinystr/Cargo.toml

```diff
@@ -64,14 +59,12 @@ harness = false
 [[bench]]
 name = "construct"
 harness = false
-required-features = ["bench"]
```

Comment:
Issue, here and maybe elsewhere: I think this code is now causing benches such as this one to be unconditionally invoked, since there isn't another place where you're hiding them. Previously, `--features bench` was required.